plugins/woocommerce-admin/client/order-attribution/style.scss

```diff
@@ -0,0 +1,61 @@
+.woocommerce-order-attribution-install-analytics-overview-banner {
+
+	margin: 0 15px 10px 0;
+	animation: isLoaded;
+	min-width: 550px; // for smaller screens
+	animation-duration: 250ms;
+
+	&.components-card {
+		box-shadow: none;
+		border: 1px solid $table-border;
+		border-radius: 8px;
+	}
+
+	.woocommerce-order-attribution-install-analytics-overview-banner__body {
+		display: flex;
+		align-items: center;
+	}
+
+	.woocommerce-order-attribution-install-analytics-overview-banner__image_container {
+		display: flex;
+		padding-left: 15px;
+		min-width: 200px; // for smaller screens
+	}
+
+	.woocommerce-order-attribution-install-analytics-overview-banner__text_container {
+		margin-inline: 24px;
+
+		> * {
+			margin-block: 1rem;
+		}
+
+		p {
+			margin-block: 0.1rem;
+		}
+	}
+
+	.woocommerce-order-attribution-install-analytics-overview-banner__text-chip {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		background: $studio-gray-0;
+		border-radius: 2px;
+		width: 42px;
+		height: 24px;
+
+		p {
+			color: $studio-gray-80;
+		}
+	}
+
+	.woocommerce-order-attribution-install-analytics-overview-banner__text-title {
+		color: $studio-gray-90;
+		line-height: 24px;
+		font-weight: 590;
```

Comment:
I see this is in the Figma as 590 as well. Just curious why design choose a non standard weight instead of 600 for example.

Reply:
Nice catch, changed it to 600 as it's no harm.
https://github.com/woocommerce/woocommerce/pull/51672/commits/749fb4dd2a5a4e34933b14b888380db0bc8cf3b3#diff-71e16bc648210308ae9a1f2e30c8e8e311fe24425077449417a2f94ca8b2aa9bR54